storage/remote/queue_manager.go

```diff
@@ -1635,8 +1640,8 @@ func (s *shards) sendSamplesWithBackoff(ctx context.Context, samples []prompb.Ti
 	return err
 }
 
-func sendWriteRequestWithBackoff(ctx context.Context, cfg config.QueueConfig, l log.Logger, attempt func(int) error, onRetry func()) error {
-	backoff := cfg.MinBackoff
+func (qm *QueueManager) sendWriteRequestWithBackoff(ctx context.Context, attempt func(int) error, onRetry func()) error {
```

Comment:
Linter is complaining that you named it `qm` here when everywhere else it's `t`. If you don't want to rename them all to `qm` it can just be `t`, I can do the clean up another time since `qm` really does make more sense.